tests/mappers/test_distcp_mapper.py

```diff
@@ -122,7 +123,7 @@ def test_setup(self):
 
     def test_task_and_relations_no_prepare(self):
         # Given
-        mapper, name = _get_distcp_mapper(
+        mapper, _ = _get_distcp_mapper(
```

Comment:
I think we can change the signature of return value and return mapper rather than tuple (same fix should be then done in the previous test then)